docs: add example to `base` tip in deploying (#994)
vuejs/vitepress: docs/guide/deploying.md

::: tip

> If your site is to be served at a subdirectory (`https://example.com/subdir/`), then you have to set `'/subdir/'` as the [`base`](../config/app-configs#base) in your `docs/.vitepress/config.js`. **Example:** If you're using Github (or GitLab) Pages and deploying to `https://<USERNAME>.github.io/<REPO>/`, then set your `base` to `/<REPO>/`.

Author: ```suggestion
If your site is to be served at a subdirectory (`https://example.com/subdir/`), then you have to set `'/subdir/'` as the [`base`](../config/app-configs#base) in your `docs/.vitepress/config.js`.

**Example:** If you're using Github (or GitLab) Pages and deploying to `https://<USERNAME>.github.io/<REPO>/`, then set your `base` to `/<REPO>/`.
```

If your site is to be served at a subdirectory (`https://example.com/subdir/`), then you have to set `'/subdir/'` as the [`base`](../config/app-configs#base) in your `docs/.vitepress/config.js`.

> **Example:** If you're using Github (or GitLab) Pages and deploying to `https://<USERNAME>.github.io/<REPO>/`, then set your `base` to `/<REPO>/`.

Author: ```suggestion
**Example:** If you're using Github (or GitLab) Pages and deploying to `https://<USERNAME or ORG>.github.io/<REPO>/`, then set your `base` to `/<REPO>/`.
```

> **Example:** If you're using Github (or GitLab) Pages and deploying to `https://<USERNAME or ORG>.github.io/<REPO>/`, then set your `base` to `/<REPO>/`.

Author: ```suggestion
**Example:** If you're using Github (or GitLab) Pages and deploying to `user.github.io/repo/`, then set your `base` to `/repo/`.
```
